src/widgets/alsa_sound.rs

```diff
@@ -0,0 +1,138 @@
+
```

Comment:
Could we make a widgets/audio module and put alsa and pulseaudio in there, so the modules are at `widgets::audio::Alsa` and `widgets::audio::PulseAudio`? That would be nice. :)